akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -293,11 +372,12 @@ final class ClusterClient(settings: ClusterClientSettings) extends Actor with Ac
           contacts = contactPoints.map(context.actorSelection)
```

Comment:
I'll also retain as a `Set`, specifically a `ListSet` so that we maintain the ordering as per the existing `Seq`. This'll save on converting to sets also.